components/homme/src/common_movie_mod.F90

```diff
@@ -3,6 +3,7 @@
 #endif
 
 module common_movie_mod
+#ifndef HOMME_WITHOUT_PIOLIBRARY
```

Comment:
In my opinion, it's better to put all the ifdefs here at the subroutine level and let the callers call these functions as usual. That way, caller code doesn't get cluttered with ifdefs. Ifdef at the tightest point possible, thus minimizing the number (ideally to 0) in non-I/O-implementation code.